mlir/include/Quantum/Transforms/Passes.td

```diff
@@ -71,4 +71,15 @@ def AdjointLoweringPass : Pass<"adjoint-lowering"> {
     let constructor = "catalyst::createAdjointLoweringPass()";
 }
 
+def RemoveChainedSelfInversePass : Pass<"remove-chained-self-inverse"> {
+    let summary = "Perform removal of chained operations that are self inverses.";
+
+    let dependentDialects = [
+        "LLVM::LLVMDialect",
+        "catalyst::CatalystDialect",
+    ];
```

Comment:
Dependent dialects only need to include those dialects from which a pass creates operations. In this case I don't think any llvm or catalyst ops are created.